lua/nvim-treesitter/parsers.lua

```diff
@@ -208,6 +208,15 @@ list.blueprint = {
   experimental = true,
 }
 
+list.bosque = {
+  install_info = {
+    url = "https://github.com/Karidus-423/tree-sitter-bosque",
+    files = { "src/parser.c" },
+  },
+  maintainers = { "@Karidus-423" },
+  experimental = true,
```

Comment:
Oh, and we no longer accept experimental parsers. Please let it cook in your repo until you are confident that it is ready for productive use and doesn't require breaking changes in the near future.

Reply:
You can adapt the snippet in the README to show (in your README) how users can install the parser and queries straight from your repo.